.github/CONTRIBUTING.md

```diff
@@ -169,12 +169,32 @@ See [Contributing to documentation related to PowerShell](#contributing-to-docum
 
 #### Pull Request - Code Review
 
-* After a successful test pass,
-  the area maintainers will do a code review,
+* Author, Reviewer, and Assignee of a PR
+  * Reviewer and Assignee are two separate roles of a PR.
+    * A Reviewer can be anyone from the collaborators.
```

Comment:
When clicking the "Reviewer" field, you can select from a list of people, that list is collaborators, right?
I think the author has the freedom to choose anyone from them. It's Assignee's duty to decide if the review is sufficient. If not, the assignee can add more reviewers.